.circleci/config.yml

```diff
@@ -351,10 +351,10 @@ workflows:
                   name: << matrix.system >>-<< matrix.color >>-<< matrix.scale >>-<< matrix.dir >>
                   matrix:
                       parameters:
-                          system: [spectrum, express]
-                          color: [light, dark]
-                          scale: [medium, large]
-                          dir: [ltr, rtl]
+                          system: [spectrum, express, spectrum-two]
+                          color: [light]
+                          scale: [medium]
+                          dir: [ltr]
```

Comment:
i would keep RTL and remove dark and large

Reply:
I would not mind, but that would result in a loss of coverage. My proposal better combines the parameters to still cover all individual options (themes, colors, scales, directions) without unnecessary duplication.